.travis.yml

```diff
@@ -64,6 +64,13 @@ install:
   - pip install google-cloud
   - pip install Pillow
 
+
+before_script:
+    # stop the build if there are Python syntax errors or undefined names
+    if [[ "$TENSORFLOW_V" == "1.8.0" ]]; then
```

Comment:
is this intended to make sure this test only runs on one build? It currently runs on 2/4 builds

Reply:
The very first line of the commit message says that we want to run flake8 on __both Python 2 and Python 3__.  This is because Abstract Syntax Trees are different for the two and so they give different results.  See the [very important note at the top of the flake8 docs](http://flake8.pycqa.org/en/latest).

* __print "hi"__ is a syntax error in Python 3 but is valid Python 2.
* __async def my_func():__ is a syntax error in Python to but is valid Python 3

Flake8 knows how to find and flag them both.

Reply:
It's probably worth adding a comment like
` # This installs a version of flake8 specific to the default python interpreter version`
or to change the install command to be explicit about which version to install, so that long term maintains of this file can see more clearly what's going on.